### PR TITLE
Distribution mapper

### PR DESCRIPTION
**Pull request by matoval**
Changed releaseMapper to match design mockup and used it to make distribution column more human readable.

### CODE REVIEW

**Review by boaz0 — COMMENTED** [review]
Hey @matoval , can you provide more information why this change is necessary?
Thanks.

**Comment by matoval** [comment]
We are using this mapper for both the image wizard and the manage images table. In the image wizard we are using the longer value and in the table we are using the shorter value.

**Comment by matoval** [comment]
I just realized this isn't going to solve the issue in the wizard. I will push a new commit with the correct solution.

File: src/Routes/ImageManager/Images.js [hunk 1]
```diff
@@ -382,7 +382,7 @@ const Images = () => {
                               },
                               item?.Version,
                               {
-                                title: releaseMapper[item?.Distribution],
+                                title: releaseMapper[`${item?.Distribution}-short`],
```
Comment (boaz0) [inline src/Routes/ImageManager/Images.js:385]: Revert this change and use `distributionMapper`:
```js
  title: distributionMapper[item?.Distribution],
```

File: src/Routes/ImageManager/steps/imageOutput.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ export default {
         value: release,
         label: releaseLabel,
       })),
-      initialValue: 'rhel-8',
+      initialValue: 'rhel-84',
```
Comment (boaz0) [inline src/Routes/ImageManager/steps/imageOutput.js:28]: create a new constant called DEFAULT_RELEASE and use it here:
```js
const DEFAULT_RELEASE ='rhel-84';
....
initialValue: DEFAULT_RELEASE
```

File: src/Routes/ImageManagerDetail/constants.js [hunk 3]
```diff
@@ -30,8 +30,8 @@ export const imageStatusMapper = {
 };
 
 export const releaseMapper = {
-  'rhel-8': 'Red Hat Enterprise Linux (RHEL) 8.4',
-  'rhel-84': 'RHEL 8.4',
+  'rhel-84': 'Red Hat Enterprise Linux (RHEL) 8.4',
+  'rhel-84-short': 'RHEL 8.4',
```
Comment (boaz0) [inline src/Routes/ImageManagerDetail/constants.js:34]: Remove this one. And create a new mapper called `distributionMapper`:
```js
export const distributionMapper = {
  'rhel-84': 'RHEL 8.4',
}

export const releaseMapper = {
  'rhel-84': 'Red Hat Enterprise Linux (RHEL) 8.4',
}
```

**Review by boaz0 — CHANGES_REQUESTED** [review]
Small changes but all in all LGTM

**Review by boaz0 — APPROVED** [review]
Flawless LGTM :100:

**Pull request merged**